integration-tests/src/test/java/com/netflix/spinnaker/clouddriver/kubernetes/it/requests/DeployManifestRequest.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2020 Armory
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.kubernetes.it.requests;
+
+import static io.restassured.RestAssured.given;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.netflix.spinnaker.clouddriver.kubernetes.it.helpers.TestManifest;
+import io.restassured.response.Response;
+import java.util.*;
+
+public class DeployManifestRequest extends BaseRequest<String> {
+
+  public Map<String, Object> deployManifest = new HashMap<>();
```

Comment:
Is this meant to simulate an instance of [`KubernetesDeployManifestDescription`](https://github.com/spinnaker/clouddriver/blob/master/clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/description/manifest/KubernetesDeployManifestDescription.java#L29)? Any reason we are not using that class directly?

Reply:
This was done intentionally, the reasoning is that by not relying on the existing implementation classes, test code is more resilient to any refactors or changes in the internal implementation. If the contracts of clouddriver with external dependencies doesn't change, then integration tests doesn't need to change either. Another benefit is that if someone changes one of these contracts inadvertently, then having to update integration tests will make it more clear that probably orca would need to also be updated.